docker-compose.yml

```diff
@@ -7,4 +7,4 @@ services:
       - .:/app:delegated
     ports:
       - "8000:8000"
-    command: npm run develop --verbose -- -H 0.0.0.0
```

Comment:
Is removing the `--verbose` here necessary? It might be important on the Pages.gov side for their logging.

Reply:
Oh thanks for noticing that, I had copy/pasted the original cloud pages example again over to start fresh, I'll add that in